spec/unit/libraries/resource_provider_spec.rb

```diff
@@ -325,7 +325,7 @@ class CatastrophicProvider < OneviewCookbook::ResourceProvider
 
     before do
       allow(res.item).to receive(:retrieve!).and_return(true)
-      allow(res.context).to receive(:scopes).and_return(['Scope1', 'Scope2'])
+      allow(res.context.instance_variable_get(:@new_resource)).to receive(:scopes).and_return(['Scope1', 'Scope2'])
```

Comment:
Why not simply `res.context.new_resource` ?

Reply:
I will use it, I didn't know that I has the same result using it.